node/src/components/block_proposer.rs

```diff
@@ -413,14 +414,14 @@ impl BlockProposerReady {
         let max_block_size_bytes = deploy_config.max_block_size as usize;
         let block_gas_limit = Gas::from(deploy_config.block_gas_limit);
 
-        let mut transfers = Vec::new();
-        let mut wasm_deploys = Vec::new();
+        let mut transfer_hashes = Vec::new();
+        let mut deploy_hashes = Vec::new();
```

Comment:
I mean, they're both deploys really.

Reply:
They are hashes, not full deploys/transfers.
Or do you mean that a "transfer" is also a "deploy"?
I made this change to be in line with the `BlockBody`, where we also use these identifiers. Unfortunately we're a bit inconsistent: In some places transfers are considered to be deploys, in other places "deploy" refers to only the Wasm ones.

Reply:
> Or do you mean that a "transfer" is also a "deploy"?

Yes, this.

Reply:
I'd also like to address this inconsistency, but that's going to be a bigger PR.
And it needs discussion first: Is a "transfer" a "deploy" or isn't it? And what do we call a non-transfer deploy then, or what do we use as an umbrella term?
But I want to avoid just introducing a third kind of terminology and make everything even less consistent.